elrond-wasm-debug/src/meta/meta_main.rs

```diff
@@ -0,0 +1,28 @@
+use elrond_wasm::contract_base::ContractAbiProvider;
+use std::env;
+
+use super::meta_config::MetaConfig;
+
+pub fn perform<AbiObj: ContractAbiProvider>() {
+    let original_contract_abi = <AbiObj as ContractAbiProvider>::abi();
+    super::meta_validate_abi::validate_abi(&original_contract_abi).unwrap();
+
+    let args: Vec<String> = env::args().collect();
+    let meta_config = MetaConfig::create(&original_contract_abi, args.as_slice());
```

Comment:
Seems to me like the `perform` function uses the second argument (first was the current executable path iirc), while the `MetaConfig::create` function only uses the ones after, so I suggest only passing `args.as_slice()[2..]` or similar.

Reply:
We are parsing them anyway, one by one, so some extra arguments slipping through are not an issue. Somehow I thought it's more flexible if it has access to all of them. Works both ways though